lib/array.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+  def sort_by_name
```

Comment:
Nice refactoring, but the accented characters aren't handled well.

Maybe we could use here something like [this](https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-transliterate) to correctly sort names.

Reply:
I think if we use the UTF-8 encoding then the I18n.transliterate is not necessary.
I commented there too, but if we decide to use transliterate then I will create a new PR.